modules/distr-tracing-otel-config-collector.adoc

```diff
@@ -0,0 +1,389 @@
+== OpenTelemetry Collector components
+
+=== Receivers
+
+==== OTLP Receiver
+
+The OTLP receiver ingests data using the OpenTelemetry protocol (OTLP).
+
+* Support level: link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]
+* Supported signals: traces, metrics
+
+.OpenTelemetry Collector custom resource with an enabled OTLP receiver
+[source,yaml]
+----
+  config: |
+    receivers:
+      otlp:
+        protocols:
+          grpc:
+            endpoint: 0.0.0.0:4317 <1>
+          http:
+            endpoint: 0.0.0.0:4318 <2>
+          tls: <3>
+            ca_file: ca.pem
+            cert_file: cert.pem
+            key_file: key.pem
+
+    service:
+      pipelines:
+        traces:
+          receivers: [otlp]
+        metrics:
+          receivers: [otlp]
+----
+<1> The OTLP gRPC endpoint. If omitted, the default `+0.0.0.0:4317+` is used.
+<2> The OTLP HTTP endpoint. If omitted, the default `+0.0.0.0:4318+` is used.
+<3> The TLS server side configuration. Defines paths to TLS certificates. If omitted TLS is disabled.
+
+==== Jaeger Receiver
+
+The Jaeger receiver ingests data in Jaeger formats.
+
+* Support level: link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]
+* Supported signals: traces
+
+.OpenTelemetry Collector custom resource with an enabled Jaeger receiver
+[source,yaml]
+----
+  config: |
+    receivers:
+      jaeger:
+        protocols:
+          grpc:
+            endpoint: 0.0.0.0:14250 <1>
+          thrift_http:
+            endpoint: 0.0.0.0:14268 <2>
+          thrift_compact:
+            endpoint: 0.0.0.0:6831 <3>
+          thrift_binary:
+            endpoint: 0.0.0.0:6832 <4>
+          tls: <5>
+
+    service:
+      pipelines:
+        traces:
+          receivers: [jaeger]
+----
+<1> The Jaeger gRPC endpoint. If omitted, the default `+0.0.0.0:14250+` is used.
+<2> The Jaeger Thrift HTTP endpoint. If omitted, the default `+0.0.0.0:14268+` is used.
+<3> The Jaeger Thrift Compact endpoint. If omitted, the default `+0.0.0.0:6831+` is used.
+<4> The Jaeger Thrift Binary endpoint. If omitted, the default `+0.0.0.0:6832+` is used.
+<5> The TLS server side configuration. See the OTLP receiver configuration section for more details.
+
+==== Zipkin Receiver
+
+The Zipkin receiver ingests data in the Zipkin v1 and v2 formats.
+
+* Support level: link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]
+* Supported signals: traces
+
+.OpenTelemetry Collector custom resource with enabled Zipkin receiver
+[source,yaml]
+----
+  config: |
+    receivers:
+      zipkin:
+        endpoint: 0.0.0.0:9411 <1>
+        tls: <2>
+
+    service:
+      pipelines:
+        traces:
+          receivers: [zipkin]
+----
+<1> The Zipkin HTTP endpoint. If omitted, the default `+0.0.0.0:9411+` is used.
+<2> The TLS server side configuration. See the OTLP receiver configuration section for more details.
+
+
+=== Processors
+
+==== Resource Detection processor
```

Comment:
[Anchoring in module files](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-in-module-files)
Missing anchor IDs